Web/Pages/Student/Student_List.cshtml

```diff
@@ -0,0 +1,214 @@
+﻿@page
+@model Web.Pages.Student.Student_ListModel
+@inject IViewLocalizer Localizer
+@{
+    ViewData["Title"] = "Student Position Results";
+    var isEnglish = System.Threading.Thread.CurrentThread.CurrentCulture.TwoLetterISOLanguageName == "en";
+}
+
+<div class="col-md-12 mrgn-tp-md mrgn-bttm-md">
+    <a href="#print"><span class="glyphicon glyphicon-print pull-right mrgn-lft-md" alt="Print document"></span> </a>
+    <a href="#save"><span class="glyphicon glyphicon-save pull-right" alt="Save document"></span> </a>
+</div>
+
+<div class="mrgn-tp-lg">
+    <table class="table table-hover table-responsive table-bordered">
+        <thead>
+            <tr>
+                <th class="bg-primary" colspan="2">@Localizer["position"]: SC-DED 02 Deckhand</th>
+            </tr>
+        </thead>
+        <tbody>
+
+            <tr><th colspan="2" style="background-color:#f2f2f2">@Localizer["certificates"]</th></tr>
+            <tr><td colspan="2">Restricted Operator Certificate-Maritime Commerical (ROC-MC)</td></tr>
+            <tr><td colspan="2">Marine Emergency Duties - Basic Safety</td></tr>
+            <tr><td colspan="2">Marine Emergency Duties - Survival Craft / Rescue Boats</td></tr>
+            <tr><td colspan="2">Marine Emergency Duties - Proficiency in Advanced Firefighting</td></tr>
+            <tr><td colspan="2">First Aid</td></tr>
+            <tr><td colspan="2">Marine First Aid</td></tr>
+            <tr><td colspan="2">Advanced Marine First Aid</td></tr>
+            <tr><td colspan="2">Bridge Watch Certificate </td></tr>
+            <tr>
+                <th style="background-color:#f2f2f2">@Localizer["knowledge"]</th>
+                <th class="text-center" style="background-color:#f2f2f2">@Localizer["level"]</th>
+            </tr>
+            <tr>
+                <td>
+                    <a href="#1" aria-controls="centred-popup" class="wb-lbx" role="button">Organizational Knowledge</a>
+                    <section id="1" class="mfp-hide modal-dialog modal-content overlay-def">
```

Comment:
Text not in resource file

Reply:
Yes, these are mockup pages. Not realizing it was outside of the Mockup directory